apps/ui/src/views/SpaceUser.vue

```diff
@@ -0,0 +1,232 @@
+<script setup lang="ts">
+import { addressValidator as isValidAddress } from '@/helpers/validation';
+import {
+  _vp,
+  autoLinkText,
+  getCacheHash,
+  getSocialNetworksLink,
+  shortenAddress
+} from '@/helpers/utils';
+import { getNetwork, supportsNullCurrent } from '@/networks';
+import type { Space, UserActivity } from '@/types';
+import type { VotingPower, VotingPowerStatus } from '@/networks/types';
+
+const props = defineProps<{ space: Space }>();
+
+const route = useRoute();
+const usersStore = useUsersStore();
+const spacesStore = useSpacesStore();
+const { param } = useRouteParser('id');
+const { resolved, address, networkId } = useResolve(param);
+const { setTitle } = useTitle();
+const { getCurrent } = useMetaStore();
+
+const userActivity = ref<UserActivity>({ vote_count: 0, proposal_count: 0 } as UserActivity);
+const loaded = ref(false);
+const votingPowers = ref([] as VotingPower[]);
+const votingPowerStatus = ref<VotingPowerStatus>('loading');
+const delegatesCount = ref(0);
+
+const network = computed(() => getNetwork(props.space.network));
+
+const userId = computed(() => route.params.user as string);
+
+const user = computed(() => usersStore.getUser(userId.value));
+
+const socials = computed(() => getSocialNetworksLink(user.value));
+
+const shareMsg = computed(() => encodeURIComponent(window.location.href));
+
+const cb = computed(() => getCacheHash(user.value?.avatar));
+
+const formattedVotingPower = computed(() => {
+  const votingPower = votingPowers.value.reduce((acc, b) => acc + b.value, 0n);
+  const decimals = Math.max(...votingPowers.value.map(votingPower => votingPower.decimals), 0);
+
+  const value = _vp(Number(votingPower) / 10 ** decimals);
+
+  if (props.space.voting_power_symbol) {
+    return `${value} ${props.space.voting_power_symbol}`;
+  }
+
+  return value;
+});
+
+const navigation = computed(() => [
+  { label: 'Statement', route: 'space-user-statement' },
+  { label: 'Delegators', route: 'space-user-delegators', count: delegatesCount.value },
+  { label: 'Proposals', route: 'space-user-proposals', count: userActivity.value?.proposal_count },
+  { label: 'Latest votes', route: 'space-user-votes', count: userActivity.value?.vote_count }
+]);
+
+async function loadUserActivity() {
+  const spaceNetwork = getNetwork(props.space.network);
+
+  const users = await spaceNetwork.api.loadLeaderboard(
+    props.space.id,
+    {
+      limit: 1,
+      skip: 0
+    },
+    'vote_count-desc',
+    userId.value
+  );
+
+  if (users[0]) userActivity.value = users[0];
```

Comment:
When i go here http://localhost:8080/#/s:aave.eth/profile/0xF1dF824419879Bb8a7E758173523F88EfB7Af193

I see `0` proposals and `565` votes
<img width="825" alt="Untitled 2" src="https://github.com/user-attachments/assets/b89ba397-2802-45f4-b0bd-db23ce32b746">

On DB I see:
<img width="802" alt="Untitled 4" src="https://github.com/user-attachments/assets/5d0fddee-8463-4f9c-a826-a778f56aa42e">

On the request input I don't see user id:
<img width="278" alt="Untitled 5" src="https://github.com/user-attachments/assets/8f7a0c4a-c715-4886-aa37-6b1ec06c969a">

Reply:
Fixed, the offchain leaderboard PR was not ready when this PR was submitted